configs/config.py

```diff
@@ -0,0 +1,23 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Optional
+
+import yaml
+
+
+@dataclass
+class Config:
+    video_directory: Path
+    video_files: Optional[List[Path]]
+    data_directory: Path
+    data_name: Optional[str]
+
+    @classmethod
+    def load_config(cls):
+        return yaml.safe_load(Path("config.yaml").read_text())
```

Comment:
this will try to load `config.yaml` from whatever the cwd is, not sure if that's intentional, but you should probably have `load_config` take a path as an argument and have some default as a fallback if none is provided. If it's intended to load a named config from within the `configs` directory then you should package that (i.e. put it within the package so it gets included when wheels are built) and make some `CONFIG_DIR` const based on `importlib.resources.files`

Reply:
hmm, I was getting ready to just use the config part as a UI (i.e host a webpage later where people can interactively type in their config), so i wasn't sure if it should be a part of the src package.

Reply:
probably in the package ya, unless you're making multiple packages in this repo - gonna need to run it somehow and `pip install cala && cala config --interactive` or something would be cool to be able to do without needing to clone repo